src/server/lib/idapi/guest.ts

```diff
@@ -0,0 +1,63 @@
+import { IdapiErrorMessages, RegistrationErrors } from '@/shared/model/Errors';
+import {
+  APIAddClientAccessToken,
+  APIPostOptions,
+  IDAPIError,
+  idapiFetch,
+} from '../IDAPIFetch';
+import { logger } from '../logger';
+import {
+  addRefToPath,
+  addRefViewIdToPath,
+  addReturnUrlToPath,
+} from '../queryParams';
+
+const handleError = ({ error, status = 500 }: IDAPIError) => {
+  if (error.status === 'error' && error.errors?.length) {
+    const err = error.errors[0];
+    const { message } = err;
+
+    switch (message) {
+      case IdapiErrorMessages.INVALID_EMAIL_ADDRESS:
```

Comment:
Only display an error message if submission to the endpoint happens without an email address (can't happen if the browser supports html5 form validation)